requirements.txt

```diff
@@ -1,18 +1,18 @@
 click >= 7.0, < 8.0
-cloudpickle >= 0.6.0, < 0.7
+cloudpickle >= 0.6.0, < 1.1
 croniter >= 0.3, < 0.4
 cryptography >= 2.2.2, < 3.0
 dask >= 0.18, < 1.2.2
 distributed >= 1.26.1, < 2.0
-docker >= 3.4.1, < 4.0
-idna < 2.8, >= 2.5
+docker >= 3.4.1, < 4.1
+idna >= 2.5, < 2.9
 marshmallow == 3.0.0b19
 marshmallow-oneofschema >= 2.0.0b2, < 3.0
-mypy >= 0.600, < 0.700
+mypy >= 0.600, < 0.800
```

Comment:
You're welcome to test out removing this for #1065

Reply:
Once this is merged I'll give that a go...